### PR TITLE
Included social authentication as feature

### PR DESCRIPTION
**Pull request by fhakamine**
The Okta Sign-In widget also supports social authentication.

## Description:
- Describe your changes. This will most likely be a copy-paste of your commit
  message(s), which should be descriptive and informative.

### Resolves:
* [Issue #X](#X)

* [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

**Comment by jmelberg-okta** [comment]
@jakubvul

**Comment by fhakamine** [comment]
approved/

File: _source/_code/javascript/okta_sign-in_widget.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ The Sign-in Widget supports the following use cases:
 
 - **Multi-Factor Authentication:** The Widget also handles enrollment and verification of multiple authentication factors. It comes with built-in support for SMS authentication, security questions, and Google Authenticator, among others.
 
+- **Social Authentication:** Support for starting social authentication with providers supported by Okta, such as Facebook Google, LinkedIn, and Microsoft.
```
Comment (jakubvul-okta) [inline _source/_code/javascript/okta_sign-in_widget.md]: Need a comma between `Facebook` and `Google`.

**Comment by jakubvul-okta** [comment]
@fhakamine please merge into the `weekly` branch, not `master`.

**Pull request closed without merging**